record-and-playback/core/scripts/archive/archive.rb

```diff
@@ -125,20 +125,27 @@ def remux_and_archive(source_dir, dest_dir)
   files .each do |file|
     ext = File.extname(file)
     next if ext.empty?
-    # These can potentially be webm (VP8/VP9), mp4 (H.264), or mkv (VP8/VP9/H.264)
-    output_basename = File.join(dest_dir, File.basename(file, ext))
-    format = {
-      extension: ext.delete_prefix('.'),
-      parameters: [
-        %w[-c copy],
-      ]
-    }
-    BigBlueButton::EDL.encode(
-      nil,
-      file,
-      format,
-      output_basename
-    )
+    begin
+      # These can potentially be webm (VP8/VP9), mp4 (H.264), or mkv (VP8/VP9/H.264)
+      output_basename = File.join(dest_dir, File.basename(file, ext))
+      format = {
+        extension: ext.delete_prefix('.'),
+        parameters: [
+          %w[-c copy],
+        ]
+      }
+      BigBlueButton::EDL.encode(
+        nil,
+        file,
+        format,
+        output_basename
+      )
+    rescue Exception => e
```

Comment:
This should rescue `StandardError`, not `Exception`. (The `Exception` base class has some special uses - like handling SIGINT - which should not be captured here)